open_humans/views.py

```diff
@@ -41,6 +39,17 @@
 TEN_MINUTES = 60 * 10
 
 
+def sort_projects_by_membership(projects):
+    """
+    Takes a queryset of projects and returns a queryset sorted by the number of
+    members in a project.
+    """
+    projects = projects.annotate(num_members=Count(
+        'project_members', filter=Q(project_members__joined=True)))
```

Comment:
Oops, I've realized that unfortunately this is still counting suspended users -- RunKeeper in particular had a fiasco that produced a large batch of suspended users. I think it'll need to add something like this...

`.annotate(num_members=Count('project_members', filter=Q(project_members__joined=True) & Q(project_members__member__user__is_active=True)))`